inc/saiport.h

```diff
@@ -47,6 +47,9 @@ typedef enum _sai_port_type_t
     /** Fabric Port */
     SAI_PORT_TYPE_FABRIC,
 
+    /** Recycle Port */
+    SAI_PORT_TYPE_RECYCLE,
+
```

Comment:
Is it more appropriate to call the SAI_PORT_TYPE_ETHERNET_RECYCLE ? This would make it clear that we are not passing meta data from the previous pass of the packet through the ASIC pipeline.

If a capability to pass meta data was to be standardized in the future we could call it a different type of recycle port.

Reply:
SAI_PORT_TYPE_RECYCLE is chosen to be generic and we could a read only attribute to get the current headers on recycle port, ETH being the default.

Reply:
This sounds good to me

Reply:
I agree with keeping the recycle port type generic and adding a separate "header mode" attribute.